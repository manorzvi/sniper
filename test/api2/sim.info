{'cmdline': ['../../run-sniper', '-n', '1', '-c', 'gainestown', '--', './api'],
 'git_revision': '',
 'host': 'butler',
 'pin_version': 'unknown',
 'rusage': (6.361482,
            0.012071,
            32952,
            0,
            0,
            0,
            6681,
            0,
            0,
            0,
            144,
            0,
            0,
            0,
            6643,
            12),
 'snipercmd': ['bash',
               '-c',
               '/home/manorz/sniper-7.4/lib/sniper -c /home/manorz/sniper-7.4/config/base.cfg --general/total_cores=1 --general/output_dir=/home/manorz/sniper-7.4/test/api2 --config=/home/manorz/sniper-7.4/config/nehalem.cfg --config=/home/manorz/sniper-7.4/config/gainestown.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpcHaZuq/run_benchmarks'],
 't_elapsed': 9.155305862426758,
 't_start': 1633334873.180484,
 'tracecmds': [['/home/manorz/sniper-7.4/record-trace',
                '-o',
                '/tmp/tmpcHaZuq/run_benchmarks',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './api']],
 'user': 'root',
 'vmem': 42508288L}
