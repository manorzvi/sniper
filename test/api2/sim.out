                                     | Core 0    
  Instructions                       |    4039381
  Cycles                             |   12108054
  IPC                                |       0.33
  Time (ns)                          |    4551900
  Idle time (ns)                     |         -3
  Idle time (%)                      |      -0.0%
Branch predictor stats               |           
  num correct                        |    1009217
  num incorrect                      |       1058
  misprediction rate                 |      0.10%
  mpki                               |       0.26
TLB Summary                          |           
  I-TLB                              |           
    num accesses                     |       4816
    num misses                       |         46
    miss rate                        |      0.96%
    mpki                             |       0.01
  D-TLB                              |           
    num accesses                     |    5009470
    num misses                       |         22
    miss rate                        |      0.00%
    mpki                             |       0.01
  L2 TLB                             |           
    num accesses                     |         68
    num misses                       |         68
    miss rate                        |    100.00%
    mpki                             |       0.02
Cache Summary                        |           
  Cache L1-I                         |           
    num cache accesses               |       4816
    num cache misses                 |        470
    miss rate                        |      9.76%
    mpki                             |       0.12
  Cache L1-D                         |           
    num cache accesses               |    5009470
    num cache misses                 |        298
    miss rate                        |      0.01%
    mpki                             |       0.07
  Cache L2                           |           
    num cache accesses               |        793
    num cache misses                 |        764
    miss rate                        |     96.34%
    mpki                             |       0.19
  Cache L3                           |           
    num cache accesses               |        789
    num cache misses                 |        764
    miss rate                        |     96.83%
    mpki                             |       0.19
DRAM summary                         |           
  num dram accesses                  |        764
  average dram access latency (ns)   |      82.95
  average dram queueing delay        |      29.52
  average dram bandwidth utilization |      0.00%
Coherency Traffic                    |           
  num loads from dram                |        163
  num loads from dram cache          |          0
  num loads from remote cache        |          0
